django/core/management/color.py

```diff
@@ -8,13 +8,20 @@
 
 from django.utils import termcolors
 
+try:
+    import colorama
+except ImportError:
+    colorama = None
+else:
+    colorama.init()
```

Comment:
Can we try moving this to a function, `has_colorama()` say? 

I know it's not much but the top-level import is a bit 🤔 — I think it might just be clearer inside a function. 
(maybe that looks horrible once we try it... — maybe then a comment would do...)

Reply:
If we move this to a function, we still need to have a global level variable to test to see if colorama is active (and thus we can turn on color, see line 24).

Also, I think moving this to a function would make it less clear what is going on. Also, this code really should only be called once (so it doesn't need to be in a function for DRY reasons). Possible replacement code:

```python

colorama = has_colorama()

def has_colorama():
     try:
        import colorama
    except ImportError:
        colorama = None
    else:
        colorama.init()

    return colorama
```